ultimatejobweb/HomeCSS.css

```diff
@@ -0,0 +1,138 @@
+* {
```

Comment:
Is it a comment? can you remove it please?

Reply:
Removed.